ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -202,6 +213,53 @@ access_lists:
         action: "< action as string >"
 ```
 
+The improved data model has a more sophisticated design documented below:
+
+```yaml
+ip_access_lists:
+  - name: "< access list name as string >"
+    counters_per_entry: < true | false >
+    entries:
+      # remark entry
+      - sequence: < acl entry sequence number >  # optional
+        # NOTE: if remark is defined, other keys in acl entry will be ignored
+        remark: "< Comment, up to 100 characters >"
+      # normal entry
+      - sequence: < acl entry sequence number >  # optional
+        action: "< permit | deny >"  # required
+        protocol: "< ip | tcp | udp | icmp | other protocol name or number >"  # required
+        # NOTE: A.B.C.D without a mask means host
+        source: "< any | A.B.C.D/E | A.B.C.D >"  # required
+        source_ports_match: "< eq | gt | lt | neq | range >"  # eq is default
```

Comment:
LGTM! just a minor suggestion. Please use our common way to set default value, ```default -> eq```.

Reply:
agree

Reply:
IMO, it can not be counted as common way until schema is releases and comment is more readable. But as discussed, it's minor. Agree and resolving.

Reply:
Reasonable!